File: Code Chef/JavaScript/500-599/592 Sasta Shark Tank SST.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const countValuation = (offer, stake) => {
    const valuation = (100 * offer) / stake;
    return valuation;
};

const findGoodDeal = (offer1, offer2) => {
    const valuationOnOffer1 = countValuation(offer1, 10);
    const valuationOnOffer2 = countValuation(offer2, 20);
    
    (valuationOnOffer1 === valuationOnOffer2)
        ? console.log('ANY')
        : (valuationOnOffer1 > valuationOnOffer2)
            ? console.log('FIRST')
            : console.log('SECOND');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [offer1, offer2] = testcase;
        findGoodDeal(offer1, offer2);
    });
});

File: Code Chef/C Programming/500-599/592 Sasta Shark Tank SST.c
#include <stdio.h>
int main() 
{
    int t,a,b;
    scanf("%d",&t);
    for(int i=0;i<t;i++)
    {
        scanf("%d%d",&a,&b);
        if(((100*a)/10)>((100*b)/20))
        {
            printf("FIRST\n");
        }
        else if(((100*a)/10)<((100*b)/20))
        {
            printf("SECOND\n");
        }
        else
        {
            printf("ANY\n");
        }
    }
	return 0;
}